test/integration/voldemort/performance/benchmark/Measurement.java

```diff
@@ -92,6 +94,16 @@ public synchronized void recordReturnCode(int code) {
         returnCodes.get(Icode)[0]++;
     }
 
+    public synchronized void recordWarningCode(int code) {
+        Integer Icode = code;

```

Comment:
I agree with you formatting suggestion and will update the code as you requested.  Originally I had just cloned the recordReturnCode method above and not thought to much about the local variable names.